tests/pygeoapi-test-config.yml

```diff
@@ -83,7 +83,7 @@ metadata:
 datasets:
     obs:
         title: Observations
-        description: Observations
+        description: Observations_description
```

Comment:
Does this update fix any breakage?

Reply:
no, unrelated to the purpose of this commit. This is just that it better validates that the content of the 'description' of the config goes to 'description' of the output